packages/@n8n/db/src/repositories/workflow-statistics.repository.ts

```diff
@@ -55,59 +55,59 @@ export class WorkflowStatisticsRepository extends Repository<WorkflowStatistics>
 		isRootExecution: boolean,
 	): Promise<StatisticsUpsertResult> {
 		const dbType = this.globalConfig.database.type;
-		const { tableName } = this.metadata;
+		const escapedTableName = this.manager.connection.driver.escape(this.metadata.tableName);
```

Comment:
This doesn't account for table prefix? Comes from before though.

Reply:
The prefix should be accounted for. What makes you think it doesn't?

Reply:
It's become a reflex when reviewing migrations, indeed `this.metadata.tableName` in a repository should be prefixed already.